api/schema/types/ProjectType.js

```diff
@@ -45,13 +45,20 @@ module.exports = gql`
             contributor_id: Int
         ): TimeSpent
         timeSpentPerContributor(
-            fromDate: String
+            fromDate: String,
             toDate: String
         ): [TimeSpent]
+        #calculated properties
+        totalAllocated(
+            fromDate: String,
+            toDate: String,
+            confirmedOnly: Boolean
+        ): Int
         totalPaid(
             fromDate: String,
             toDate: String
-        ): Int
+        ): Int # The total paid from the client, takes start date and ed date as attributes
```

Comment:
In GQL syntax it is actually better to expose these comments to the GraphQL explorer by writing as so:

```
"
"The total paid from the client, takes start date and end date as attributes"
totalPaid(
    fromDate: String,
    toDate: String
): Int
The total incurred not paid from the client
totalIncurredPayments: Int
```

Only use the `#` operator if you are putting comments in shouldn't show up in the GraphQL explorer

Also separately for this comment in particular the "takes start date and ed date as attributes" is redundant since it is already clear from the GQL specification that these are the arguments to pass

Reply:
resolved on pr #326